bpf/lib/nodeport.h

```diff
@@ -549,15 +549,8 @@ static __always_inline int nodeport_lb6(struct __ctx_buff *ctx,
 			return ret;
 	}
 
-	if (!svc || (!lb6_svc_is_loadbalancer(svc) &&
-		     !lb6_svc_is_nodeport(svc) &&
-		     !lb6_svc_is_external_ip(svc) &&
-		     !lb6_svc_is_hostport(svc))) {
-		if (svc)
-			return DROP_IS_CLUSTER_IP;
-
```

Comment:
I would probably leave all these helpers around since they improve code readability. So this one above, I'd change it into a small inline helper, something like: `lb{4,6}_svc_is_routable(svc)` . The only thing we filter out here is ClusterIP service type and this is the only non-routable type where we must drop pkts coming in.